Demo/WhisperDemo/WhisperDemo/ViewController.swift

```diff
@@ -123,24 +119,23 @@ class ViewController: UIViewController {
     Whisper(message, to: navigationController, action: .Present)
   }
 
-  func nextButtonDidPress() {
-    let controller = DetailViewController()
-    title = ""
-    navigationController?.pushViewController(controller, animated: true)
+  func presentNotificationDidPress(button: UIButton) {
+    let announcement = Announcement(title: "Ramon Gilabert", subtitle: "Vadym Markov just commented your post", image: UIImage(named: "avatar"))
+
+    Shout(announcement, to: self)
```

Comment:
Whisper and Silent are uppercase, I think it makes more sense, it's like something big! We decided to do it like that before...